server/routes/pollRouter.js

```diff
@@ -29,21 +30,32 @@ router.post('/', auth, async (req, res) => {
         });
         
         const imageOneUrl = uploadedResponseOne.secure_url;
+        const imageOneVotes = 0;
         const imageTwoUrl = uploadedResponseTwo.secure_url;
+        const imageTwoVotes = 0;
```

Comment:
since these are default values, you can move it up in this function and named it `const DEFAULT_VOTES = 0` since the value 0 is what we probably are looking for

Reply:
Resolved as per suggestion